src/test/java/teammates/test/cases/BaseTestCaseWithDatastoreAccess.java

```diff
@@ -187,6 +187,8 @@ private void equalizeIrrelevantData(AccountAttributes expected, AccountAttribute
                 expected.studentProfile.googleId = actual.googleId;
             }
             expected.studentProfile.modifiedDate = actual.studentProfile.modifiedDate;
+            // data is expected to be sanitized
+            expected.sanitizeForSaving();
```

Comment:
Data which required sanitization was added to typicalDataBundle.json, and certain `*Attributes` still have html sanitization before saving. So `expected`, which is obtained from the json, would differ from the actual data in terms of sanitization. `verifyPresentInDatastore(DataBundle data)` takes in the data bundle directly and users are unlikely to do sanitization on each entity in the bundle beforehand so this method was added in here so as to equalise these differences.